pages/dashboard.js

```diff
@@ -202,21 +205,35 @@ export async function getServerSideProps({ req, locale }) {
   // const session = await getSession({ req })
   // const isAuth = session ? true : false
 
-
   const metadata = {
     title: 'Digital Centre (en) + Digital Centre (fr)',
     keywords: 'en + fr keywords',
     description: 'en + fr description',
   }
 
   const langToggleLink = locale === 'en' ? '/fr/dashboard' : '/dashboard'
-
+  const t = locale === 'en' ? en : fr
   const usersBenefts = [] // to be retrieved by API
   const cppData = await GetCPPProgramData()
   if (cppData) usersBenefts.push(cppData)
   const eiData = await GetEIProgramData()
   if (eiData) usersBenefts.push(eiData)
 
+  const breadCrumbItems = [
+    {
+      link: t.url_canada_ca,
+      text: t.canada_ca,
+    },
+    {
+      link: t.url_serviceCanada,
+      text: t.serviceCanada,
+    },
+    {
+      link: t.url_myBenefitsAndServices,
+      text: t.myBenefitsAndServices,
```

Comment:
The current page should not be listed in the breadcrumb trail.
https://design.canada.ca/common-design-patterns/breadcrumb-trail.html